src/params/str-list.js

```diff
@@ -10,6 +10,18 @@ class StrList extends List {
       assert(opts.enums.every(e => typeof e === 'string'));
       this.enums = new Set(opts.enums);
     }
+    if (opts.minListItemLength !== undefined) {
+      assert(typeof opts.minListItemLength === 'number');
```

Comment:
This is the better check for these, we don't want to support list items of length `1.2` ;)
```suggestion
      assert(Number.isInteger(opts.minListItemLength);
```